<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post3dWindowContourGroupDataItem_SettingEditWidget</class>
 <widget class="QWidget" name="Post3dWindowContourGroupDataItem_SettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="rangeTab">
      <attribute name="title">
       <string>Faces</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="Post3dWindowFaceListSettingEditWidget" name="faceListWidget" native="true"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>464</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="colormapTab">
      <attribute name="title">
       <string>Colormap</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidgetContainer" name="colorMapWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="ContourSettingEditWidget" name="contourWidget" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="OpacityContainerWidget" name="opacityWidget" native="true"/>
     </item>
     <item>
      <widget class="QCheckBox" name="lightingCheckBox">
       <property name="text">
        <string>Lighting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TooltipLabel" name="lightingToolTopLabel">
       <property name="toolTip">
        <string>When lighting is enabled, The color of the contour changes depending of the angle against the camera.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>358</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Post3dWindowFaceListSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>post/post3d/post3dwindowfacelistsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QWidgetContainer</class>
   <extends>QWidget</extends>
   <header>misc/qwidgetcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OpacityContainerWidget</class>
   <extends>QWidget</extends>
   <header>guibase/widget/opacitycontainerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TooltipLabel</class>
   <extends>QLabel</extends>
   <header>guibase/tooltiplabel.h</header>
  </customwidget>
  <customwidget>
   <class>ContourSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/contour/contoursettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
